README.md

```diff
@@ -445,10 +445,20 @@ We have (more than) enough themes for the time being. Please add your theme to t
 
 ## Contributors
 
+<a href="https://github.com/ohmyzsh/ohmyzsh/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ohmyzsh/ohmyzsh" width="100%"/>
+</a>
+
```

Comment:
Could you move it below `Thank you so much!` line?

Reply:
I have deleted

Reply:
I mean in the end of the file. Could you move the contributor picture to the end of the file?